frontend/src/pages/Dashboard/index.tsx

```diff
@@ -11,7 +13,12 @@ function Dashboard({ getAllDashboards }: DashboardProps): JSX.Element {
 		getAllDashboards();
 	}, [getAllDashboards]);
 
-	return <ListOfAllDashboard />;
+	return (
+		<Space direction="vertical" size="middle" style={{ width: '100%' }}>
+			<ReleaseNote path={window.location.pathname} />
```

Comment:
can we grab pathname from useLocation hook rather then window object

Reply:
done